### ISSUE TITLE
Bionicle Heroes (USA) Secret locator point on map layered incorrectly

### ISSUE THREAD

**Issue by SilasLaspada** [Arisotura/melonDS#1039]
The blue secret locator point (The blue character's special ability) on the map is layered incorrectly, showing over the map background layer and some UI elements, where it should be hidden under all UI elements. I am currently looking for my DS to 100% make sure that this isn't a game bug, but it really shouldn't be.

How it normally looks:
![melonDS_G59vA1p2jG](https://user-images.githubusercontent.com/24215969/112071548-a0121800-8b35-11eb-93ac-62c655467562.png)

The bug (under the top blue hexagon):
![melonDS_YEdLGQNouR](https://user-images.githubusercontent.com/24215969/112071563-a7d1bc80-8b35-11eb-93b6-4e8b59b1111b.png)

**Comment by RSDuck** [comment]
are you using the software or the OpenGL renderer?

**Comment by SilasLaspada** [comment]
Software, would you like me to try on OpenGL?

**Comment by RSDuck** [comment]
no, if anything that could have fixed it.

Can you give us a savefile to get to get as close as possible to this part of the game?

**Comment by SilasLaspada** [comment]
Of course! First, make sure to enable unlimited ammo, invincibility, and one hit defeat in the extras menu. Next, select the zone "Volcanic Core" and the level "Cooling Vats", progress to the second room and kill all the enemies, destroy the two canisters behind the lava pillars (screenshot attached), select the middle blue character, and activate their special ability with the "R" button. The bug is easiest seen close to the exit door while facing away from it.

Canister:
![melonDS_PSr0kbjmqM](https://user-images.githubusercontent.com/24215969/112083952-9d232180-8b4d-11eb-8c28-ec547c3d26cb.png)

Bug:
![melonDS_50wcZ4UZAs](https://user-images.githubusercontent.com/24215969/112083972-a6ac8980-8b4d-11eb-8fd0-57a8afe02545.png)

[Save File.zip](https://github.com/Arisotura/melonDS/files/6186507/Save.File.zip)

**Comment by TheRogueArchivist** [comment]
Still present in 21590b070976f8fe343fdc44d4fee599cbda94f4.
